std/assembly/performance.ts

```diff
@@ -12,9 +11,13 @@ import {
 export namespace performance {
   export function now(): f64 {
     if (isDefined(ASC_WASI)) {
-      let err = clock_time_get(clockid.MONOTONIC, 1000, tempbuf); // TODO: more precision?
-      if (err) throw new Error(errnoToString(err));
-      return <f64>load<u64>(tempbuf) / 1000000;
+      let err = clock_time_get(clockid.MONOTONIC, 1000000, tempbuf);
+      if (err) {
+        // try with less precission
+        err = clock_time_get(clockid.MONOTONIC, 1000, tempbuf);
+        if (err) throw new Error("Can't retrive time from clock_time_get");
```

Comment:
Should this perhaps report the underlying error in `err`?

Reply:
`errnoToString` signifintally increase code size while set of errors is pretty limit here.

Reply:
I don't think properly handling of errors in clock wasi routines worth code size. Failure cases should be really rare. And even if we display some error code, it probably won't mean anything since it comes from the syscall api system.

Reply:
How I see it is that we are kinda unnecessarily losing information about the error cause there. If something goes wrong, then forwarding what the underlying layer knows about it is the typical thing to do. More rare, than not so different than dealing with network errors, which can have lots of causes.

Reply:
Okay. How about a script that transforms static error strings [here](https://github.com/AssemblyScript/assemblyscript/blob/main/std/assembly/bindings/wasi_snapshot_preview1.ts#L961) from UTF16 to UTF8's `mamory.data`. That we would get at least a huge switch of pointers and don't include String.UTF8.encode method?

Reply:
The reason for including UTF8.encode is not primarily the strings there, but that printing *any* AS string requires it. That these are currently always printed is merely a side-effect of immediately calling `abort` because `new Error` is not implemented. Otherwise seems like independent aspects.

Reply:
That's a problem. I quite accept that people who compile in WASI can use just one of the methods like performance.now, Math.random, Date.now but not use output or input, which leads to a huge switch and utilities to convert to utf8 would be included for very rare case (printing an error). I would like to avoid this

Reply:
Ok, I refactored `errnoToString`. And throw an error with `errnoToString` always except `ASC_NO_ASSERT`. WDYT?